_opensearch/rest-api/rank-eval.md

```diff
@@ -0,0 +1,115 @@
+---
+layout: default
+title: Ranking evaluation
+parent: REST API reference
+nav_order: 82
+---
+
+# Ranking evaluation
+
+The [rank]({{site.url}}{{site.baseurl}}/opensearch/supported-field-types/rank/) eval endpoint allows you to evaluate the quality of ranked search results.
+
+## Path and HTTP methods
+
+```
+GET <index_name>/_rank_eval 
+POST <index_name>/_rank_eval
+```
+
+## Query parameters
+
+Query parameters are optional.
+
+Parameter | Data Type | Description
+:--- | :---  | :---
+ignore_unavailable | boolean | Defaults to `false`. When set to `false` the response body will return an error if an index is closed or missing.
+allow_no_indices | boolean | Defaults to `true`. When set to `false` the response body will return an error if a wildcard expression points to an indices that are closed or missing.
+expand_wildcards | string | Expand wildcard expressions for indices that are `open`, `closed`, `hidden`, `none`, or `all`.
+search_type | enum | Set search type.
```

Comment:
Per Fanit: We can't just say 'enum': we need to say "string" and then list all options in the description.